### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by vidimus** [mmnr3/blackrose-estate#1]
Hello, @mmnr3 
Is this project alive? Do you need any help with it or any assitance with the editor / playtesting?

**Comment by mmnr3** [comment]
Yes, it is kind of alive. I need to create a more interesting ending and have time to do it. My work have been steeling my timefromthis project. :)

**Comment by vidimus** [comment]
Thanks for the quick reply, @mmnr3 
Happy to hear you want to continue it. I hope you can get some free time to complete. If you need someone for the playtesting, feel free to contact me.
Cheers!

**Comment by vidimus** [comment]
Hello, @mmnr3 , are you still working on this?

**Comment by vidimus** [comment]
Any news, @mmnr3 ?

**Comment by mmnr3** [comment]
Ah, it is on ice at the moment. I'm still searching for some good images i
can use.

Den ons 26 juli 2023 kl 00:42 skrev Vidimus ***@***.***>:

**Comment by vidimus** [comment]
Hello, @mmnr3 
Have you tried with AI generated pictues? There are some places like Leonardo AI and Womb.art. i have used  both for a scenario, and some others authors too.

**Comment by mmnr3** [comment]
Cool. I'll try them.

Den tis 1 aug. 2023 kl 10:17 skrev Vidimus ***@***.***>:

**Comment by vidimus** [comment]
Hello, any new?

**Comment by vidimus** [comment]
Hello, @mmnr3 . Are you still working on the scenario?

**Comment by vidimus** [comment]
Hello, @mmnr3 .

Any progress with this?

**Comment by mmnr3** [comment]
Hi.
Sorry no. I have had alot of work at work. But I have fixed images and
people are on me to do voiceover. :)
So I have to work on the story over christmas.

Den ons 29 nov. 2023 kl 22:37 skrev Vidimus ***@***.***>:

**Comment by vidimus** [comment]
Hello, @mmnr3 

Have a look to this AI voice generator. https://genny.lovo.ai/signin

**Comment by vidimus** [comment]
Hello, @mmnr3 , any news about this project?

**Comment by vidimus** [comment]
Hello, @mmnr3 . Any progress so far?

**Comment by mmnr3** [comment]
Hi, I'm restarting the work on it. New energy now that the sun i looking
over the horizon again. :)

Den mån 15 apr. 2024 kl 19:55 skrev Vidimus ***@***.***>:

**Comment by vidimus** [comment]
Hello, @mmnr3 
Any progress with your scenario?
Cheers!

**Comment by vidimus** [comment]
Hello, @mmnr3, how are you doing? Any news about this project?

**Comment by vidimus** [comment]
Hello, @mmnr3, something about your scenario?

**Comment by vidimus** [comment]
Hello, @mmnr3. I guess you are not working anymore in this scenario, aren't you?

**Comment by mmnr3** [comment]
I got a new job and a boat so it is further down my priority list. But I do
want to finish it sometime. 🙂

Den tors 25 sep. 2025 kl 23:19 skrev Vidimus ***@***.***>:

**Comment by vidimus** [comment]
Hello, @mmnr3 

Boat and a new job... Did you join the Navy? ;-)

Let me know in case you need any help with the scenario, in case you decide to retake the project.

Best regards,
